src/dlgTriggerEditor.cpp

```diff
@@ -138,6 +138,20 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
                        "<p>This will create a string called 'foo' with 'bar' as its value.</p>"
                        "<p>Check the manual for <a href='http://wiki.mudlet.org/w/Manual:Contents'>more information</a>.</p>");
 
+    // Descriptions for screen readers, clarify to translators that the context of "activated" is current status and not confirmation of toggle.
+    descActive = tr("activated", "Item is currently on, short enough to be spoken");
```

Comment:
Normally not a fan of centralising strings, but when there are 30+ instances of it, well :D